backend/scripts/seed_demo_data.py

```
@@ -0,0 +1,494 @@
+            session.add(t)
+            session.commit()
+            session.refresh(t)
+            if grp_name in groups:
+                session.add(TargetGroup(target_id=t.id, group_id=groups[grp_name].id))
+                session.commit()
+        targets.append(t)
+
+    # 5. SLA & Policy Rules
+    sla_configs = [(Severity.CRITICAL, 7), (Severity.HIGH, 14), (Severity.MEDIUM, 30), (Severity.LOW, 90)]
+    for sev, days in sla_configs:
+        existing = session.exec(select(SlaRule).where(SlaRule.workspace_id == ws_prod.id, SlaRule.severity == sev, SlaRule.group_id == None)).first()
```

Ruff (0.16.3)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)